tools/runners/run-test-suite.sh

```diff
@@ -168,9 +172,11 @@ do
         status_code=$?
     fi
 
+    testfile=$(basename $(echo $cmd_line | sed s/" "/"\n"/g | tail -1))
```

Comment:
I still don't get this. Isn't `$full_test` already available here?

Reply:
My bad, I didn't see the `$full_test` variable. But that variable contains the absolute path to the test file, which is really not necessary in my opinion. I think it would be best if we display only the name of the js file, or maybe the path relative to the `jerryscript` directory.

Reply:
I was most troubled by the echo+sed+tail combo. Basename might be OK. As well as the way the relative path is already being determined (see 5 lines above). So, either `$(basename $full_test)` or `${full_test#$ROOT_DIR}`.